examples/js/loaders/GLTFLoader.js

```diff
@@ -2460,19 +2827,34 @@ THREE.GLTFLoader = ( function () {
 
 			} else {
 
-				var geometryPromise;
+				var geometryPromise = this._onBefore( 'Geometry', primitive ).then( function ( primitive ) {
```

Comment:
`Geometry` should probably be `geometry`.

Reply:
Updated, thanks.